### ISSUE TITLE
Publish diffing UI to github pages

### ISSUE THREAD

**Issue by javigomez** [GoogleChrome/lighthouse-ci#131]
Hi,

Is there any chance to run two urls and do a local "combined report" (if possible, without the local server instance)?

I mean, this is my CI:

```
- build branch
- deploy branch to staging server
- lhci autorun --collect.url=http://staging.my-web.com/master --collect.url=http://staging.my-web.com/pull_request_1
- generate compared report
```

So I would like to have now a DIFF report between my Master and my Pull Request. Something like:

![Screenshot 2019-12-02 at 18 27 49](https://user-images.githubusercontent.com/1375475/69980913-80667300-1531-11ea-94a8-d2fe1682ea96.png)

I was thinking on creating my own code based on the CompareUrl feature of the lhci server. I am looking at the source code but I don't seem to find easily where this combined reports are generated. Could you point me in the right direction?

Thanks very much.

**Comment by patrickhulce** [comment]
There's not a way to do this without the server currently, but we have plans to publish the diff'ing UI on a public page that folks can use to compare arbitrary reports like you're talking about.

We'll let this issue track that effort :)

**Comment by connorjclark** [comment]
Should we do a new app, or expand the viewer to accept / render a single LHR and two? I prefer the latter, fwiw.

**Comment by patrickhulce** [comment]
Eventually, sure, but publishing the code/logic as a separate thing is step one of integrating the logic into other things :)

**Comment by patrickhulce** [comment]
Ironically I don't actually have rights to enable gh pages, but the branch is there :)

**Comment by javigomez** [comment]
Hi, @patrickhulce works like a charm!!!! thank you very much

<img width="1081" alt="Screenshot 2020-01-10 at 13 17 12" src="https://user-images.githubusercontent.com/1375475/72152531-90346b00-33ab-11ea-9f05-7dff664decc5.png">

The Drag is not working properly, it blinks when you try to drag the file, like if something else is stealing the foucus. That ends up in sometimes a proper Upload, and sometimes you drag the json to the browser, ending up in the browser opening it. This happens in Chrome and Firefox. I can add more details if needed.

**Comment by patrickhulce** [comment]
thanks @javigomez fixed by 9cd66c295f453cf60581ce619f548521c4c95cd2
